src/Home.svelte

```diff
@@ -0,0 +1,242 @@
+                <h2 class="mt-8">Unrecognized terms</h2>
+                <table>
+                  <thead class="border-b-4 border-white">
+                    {#each $output[selectedView].unknown.headers as header}
+                      <th class="text-left p-2">{header}</th>
+                    {/each}
+                  </thead>
+                  <tbody class="whitespace-nowrap">
+                    {#each $output[selectedView].unknown.body as row}
+                      <tr class="border-b-2 border-white">
+                        {#each row as column}
+                          <td class="p-2">{column}</td>
+                        {/each}
+                      </tr>
+                    {/each}
+                  </tbody>
+                </table>
+              {/if}
+              <!-- TODO: Verifiable Presentation -->
+            {:else}
+              <h2>Document</h2>
+            {/if}
+          {:else if selectedView === "rdf"}
+            <h3 class="text-left italic">Document</h3>
+            <table>
+              <thead class="border-b-4 border-white">
+                {#each $output[selectedView].document.headers as header}
+                  <th class="text-left p-2">{header}</th>
+                {/each}
+              </thead>
+              <tbody class="whitespace-nowrap">
+                {#each $output[selectedView].document.body as row}
+                  <tr class="border-b-2 border-white">
+                    {#each row as column}
+                      <td class="p-2">{column}</td>
+                    {/each}
+                  </tr>
+                {/each}
+              </tbody>
+            </table>
+            <h3 class="text-left italic mt-2">Proof Options</h3>
+            <table>
+              <thead class="border-b-4 border-white">
+                {#each $output[selectedView].proofOptions.headers as header}
+                  <th class="text-left p-2">{header}</th>
+                {/each}
+              </thead>
+              <tbody class="whitespace-nowrap">
+                {#each $output[selectedView].proofOptions.body as row}
+                  <tr class="border-b-2 border-white">
+                    {#each row as column}
+                      <td class="p-2">{column}</td>
+                    {/each}
+                  </tr>
+                {/each}
+              </tbody>
+            </table>
+          {:else}
+            <h3 class="text-left italic">Document</h3>
+            <div class="border border-white overflow-auto">
+              {#each $output[selectedView].docNquads.split("\n") as nquad}
+                {nquad}
+              {/each}
+            </div>
+            <h3 class="text-left italic mt-2">Proof Options</h3>
+            <div class="border border-white overflow-auto">
+              {#each $output[selectedView].proofNquads.split("\n") as proofNquad}
+                {proofNquad}
+              {/each}
+            </div>
```

Comment:
Newlines and spaces are significant in N-Quads.
```suggestion
            <pre class="border border-white overflow-auto">
              {$output[selectedView].docNquads}
            </pre>
            <h3 class="text-left italic mt-2">Proof Options</h3>
            <pre class="border border-white overflow-auto">
              {$output[selectedView].proofNquads}
            </pre>
```